src/core.js

```diff
@@ -549,20 +549,26 @@ $.extend( $.validator, {
 
 			// select all valid inputs inside the form (no submit or reset buttons)
 			return $( this.currentForm )
-			.find( "input, select, textarea" )
+			.find( "input, select, textarea, [contenteditable]" )
 			.not( ":submit, :reset, :image, [disabled]" )
 			.not( this.settings.ignore )
 			.filter( function() {
-				if ( !this.name && validator.settings.debug && window.console ) {
+			    var name = this.name || $(this).attr("name"); // for contenteditable
```

Comment:
Please use tabs instead of spaces. Also, please add one space after `(` and one before `)` in `$(this).attr("name")`